nx-X11/programs/Xserver/hw/nxagent/Error.c

```diff
@@ -372,68 +372,63 @@ char *nxagentGetHomePath(void)
   return nxagentHomeDir;
 }
 
+/*
+ * returns a pointer to the static nxagentRootDir. The caller must not free
+ * this pointer!
+ */
```

Comment:
Where do you free nxagentRootDir, finally? Leaving it to the application cleanup at end of runtime?

Reply:
yes, same for nxagentHomeDir.